test/helpers/index.ts

```diff
@@ -71,3 +74,37 @@ export const setupTestEnvironment = createFixture(
     };
   }
 );
+
+export const createRemovalTokenId = async (
+  removalInstance: Removal,
+  options?: Partial<UnpackedRemovalIdV0Struct>
+): Promise<BigNumber> => {
+  const defaultRemovalData: UnpackedRemovalIdV0Struct = {
```

Comment:
not sure a default value is really that useful here

Reply:
I think I disagree.. otherwise we'd have to pass values for all of these things everywhere we use this utility, when we don't actually care what the value of most of them is